### ISSUE TITLE
Delegate `clearFolder` to library user

### ISSUE THREAD

**Issue by lzambarda** [askiada/external-sort#5]
Just an idea, I think https://github.com/askiada/external-sort/blob/8a71574cb0791a63b202f639cf78e754bd4aa106/file/file.go#L42 is potentially dangerous.
What if the responsibility was completely offloaded to the user of the library?

**Comment by askiada** [comment]
Good point! The processing is creating chunk files. And we don't want to keep these when we are done. Also we don't want to process a potential chunk file from a previous run. I don't know what would be a better approach. Maybe to only remove files with chunk_*.csv pattern. So at least you don't remove other user files

**Comment by lzambarda** [comment]
I think that is a good alternative!
This could even be driven by a parameter or something, but prob a pattern is good enough with no extra development needed!
